vault/blog.2021.2021-12-08-dendron-with-netlify.md

```diff
@@ -0,0 +1,97 @@
+---
+id: 7h7zZkjF4Yqz8XSrHS1je
+title: "Share Your Notes Online: Publish Dendron with Netlify and GitHub"
+desc: ''
+updated: 1638395819536
+created: 1638387034460
+image:
+  url: >-
+    https://org-dendron-public-assets.s3.amazonaws.com/images/blog-header-dendron-netlify.png
+  alt: Visual Studio Code logo
+date: '2021-12-08'
+excerpt: >-
+  Publishing Dendron notes, using Netlify and GitHub, can be done quickly with the Dendron publishing template!
+author: Derek Ardolf
+published: false
+---
+
+![Logos for GitHub, Netlify, and Dendron](https://org-dendron-public-assets.s3.amazonaws.com/images/blog-header-dendron-netlify.png)
+
+[Dendron](https://www.dendron.so/) helps people and products organize with notes, docs, and knowledge of all the things. Let's see how easy it is to publish and have a static site hosted on [Netlify](https://www.netlify.com/)!
+
+You'll learn how to publish your Dendron notes using Netlify and GitHub, which looks like [this demo website](https://dendron-example.netlify.app/). This tutorial assumes that you have basic knowledge of GitHub and Visual Studio Code (VSCode).
+
+- [Join GitHub](https://github.com/join)
+  - [Learn: Git and GitHub](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/GitHub)
+- [Join Netlify](https://app.netlify.com/signup)
+- [Download Visual Studio Code](https://code.visualstudio.com/)
+
+> This uses a brand new Dendron Workspace you can later configure to work with your own vaults. Do you already have a workspace you'd prefer using? Reference the Dendron docs for in-depth details on how the template, and other workspaces, can publish to Netlify: [Publishing with Netlify](https://wiki.dendron.so/notes/yetuum6o9wZi6eVJQBbQb).
+
+## Part 1: GitHub
+
+When logged into GitHub, [click here](https://github.com/dendronhq/template.publish.netlify/generate) to generate a new Dendron Workspace pre-configured for Netlify publishing. If you're wondering, we're using [this template](https://github.com/dendronhq/template.publish.netlify) on GitHub.
+
+![Generating a new GitHub repo from the Dendron Netlify template](https://org-dendron-public-assets.s3.amazonaws.com/images/github-create-workspace-netlify.gif)
+
+Nice! That's all we need to do for this minimal Dendron setup.
```

Comment:
- can you add a detail section that describes what the github template is doing?
    - that could be in the `README.md` of the template and we can link to it from the blog post
        > to understand what is going on behind the scenes, you can read the README